### PR TITLE
batch copy permission on module

### PR DESCRIPTION
**Pull request by alikon**
follow up #24730 for module batch copy.

### Summary of Changes
let's copy permission too

### Testing Instructions

-   Create/edit a module.
-   Go to its permission tab and make a change. (e.g registered->allow creating)
-   Go to Batch and Copy it somewhere else.

### Expected result
the permission are copyied too

### Actual result
permission are not copied

### CODE REVIEW

File: administrator/components/com_modules/models/module.php [hunk 1]
```diff
@@ -201,6 +204,17 @@ protected function batchCopy($value, $pks, $contexts)
 					$db->setQuery($query);
 					$db->execute();
 				}
+
+				// Copy rules
+				$query->clear()
+					->update($db->quoteName('#__assets', 't'))
+					->join('INNER', $db->quoteName('#__assets', 's') .
+					 ' ON ' . $db->quoteName('s.id') . ' = ' . $oldAssetId
```
Comment (richard67) [inline administrator/components/com_modules/models/module.php]: Drone says there is a space instead of a tab in line 212.

**Review by richard67 — CHANGES_REQUESTED** [review]

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 8f074f3c2dc0a5d4badbf148470e6579b1f2cc3b<br /><br />I've tested with 2 modules which had differenc ACL modifications.
Both modules copied together in 1 batch action.
Before patch: Copied modules have ACL like in new modules.
After patch: Each of the module copies has ACL like the particular copy source module.

Drone error (javascript) seems not to be related to this PR. Code style is ok.

@HLeithner and @wilsonge Release leads please check as also with PR #24730 and PR #24736 if this is a bug fix so it goes into 3.9.next or a new feature for 4.0.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24737">issues.joomla.org/tracker/joomla-cms/24737</a>.</sub>

File: administrator/components/com_modules/models/module.php (hunk 1 above)
Comment (alikon) [reply administrator/components/com_modules/models/module.php]: should be fixed now
now drone complain about javascript :no_good_man:

**Review by richard67 — APPROVED** [review]

File: administrator/components/com_modules/models/module.php (hunk 1 above)
Comment (richard67) [reply administrator/components/com_modules/models/module.php]: Drone error now seems to me to be not to be related to this PR.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 8f074f3c2dc0a5d4badbf148470e6579b1f2cc3b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24737">issues.joomla.org/tracker/joomla-cms/24737</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by infograf768** [comment]
relaunch drone

**Comment by richard67** [comment]
@franz-wohlkoenig Any reason why this is not merged? Has 2 good tests, has RTC. Only thing is it needs a "rebase" (i.e. merge from staging).

**Comment by infograf768** [comment]
Rebased. @HLeithner please decide if merging in 3.9

**Comment by getimo** [comment]
Any progress on that?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24737">issues.joomla.org/tracker/joomla-cms/24737</a>.</sub>

**Pull request merged**